Conch/lib/Conch/Controller/DeviceLocation.pm

```diff
@@ -29,7 +29,7 @@ sub get ($c) {
 		{ error => "Device $device_id is not assigned to a rack" } )
 		unless $maybe_location;
 
-	$c->status( 200, $maybe_location->as_v1_json );
+	$c->status( 200, $maybe_location->TO_JSON );
```

Comment:
You shouldn't need that direct call. Mojo will call TO_JSON for you when it's appropriate.

Reply:
Yup. I did a global find-and-replace for the string and intentionally didn't do any functional refactors like this. Do you want me to include this change in this PR, or have a separate?

Reply:
Let's go ahead and include it in this PR and just do the transition all at once